tools/ci_build/github/linux/docker/scripts/install_centos.sh

```diff
@@ -21,12 +21,14 @@ elif [ "$os_major_version" == "6" ] && [ ! -d "/opt/python/cp35-cp35m" ]; then
   /usr/bin/python3.6 -m pip install --upgrade pip
 else
   yum install -y redhat-lsb-core expat-devel libcurl-devel tar unzip curl zlib-devel make libunwind icu aria2 rsync bzip2 git bzip2-devel
-  # install dotnet core dependencies
-  yum install -y lttng-ust openssl-libs krb5-libs libicu libuuid
-  # install dotnet runtimes
-  yum install -y https://packages.microsoft.com/config/centos/7/packages-microsoft-prod.rpm
-  yum install -y dotnet-sdk-2.2
 
+  if [ "$os_major_version" == "7" ]; then
```

Comment:
>f [ "$os_major_version" == "7" ]; then [](start = 3, length = 38)

This used to be a default case but now for some OS nothing will get installed and no message is being displayed.

Reply:
we want to install dotnet dependencies and sdk which are inside this if loop only on centos 7... this is intentional

I added the dotnet sdk and dependency installation in my previous commit with the very same intention. I made a wrong assumption that only cases with os version 7 will fall in this IF loop... now using this PR to fix it